pages/api/helpjuice/api/questions/popular.page.ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import fetch from 'node-fetch';
+
+const proxyPopularQuestions = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+): Promise<void> => {
+  try {
+    const helpjuiceUrl = process.env.HELPJUICE_CATEGORY_ID
```

Comment:
is helpjuice 1 word or 2? Should it be written like `helpJuice`

Reply:
I've only seen it as one word.

Reply:
I've only seen it as one word, but it looks weird. Not something we need to amend in this PR.